cogs/numb.py

```diff
@@ -0,0 +1,63 @@
+'''This is a guess the number game'''
+import random
+import asyncio
+from discord.ext import commands
+
+
+GuessesTaken = 0
```

Comment:
This seems like a local variable.  Please keep it internal to your class, or better yet, your `numguess` method.